.github/workflows/branches.yml

```diff
@@ -12,56 +12,36 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - id: cache-packages-volumes
```

Comment:
This was an old attempt at caching in github actions to improve circle ci performance.  It was simply WAY too complicated for the value it was adding.  The new approach is simpler: explicitly build images that cache from existing ones in dockerhub that we push at the end of our build process.  There will be some downloading overhead, but this downloading will be bundled, composed, and smaller than the individual processes that were running before.